training/platform-services/PLATFORM.md

```diff
@@ -0,0 +1,182 @@
+Platform Services
+===========================
+
+Platform services allow us to develop, run, and manage applications without the complexity of building and maintaining the infrastructure typically associated with developing and launching an app. This documentation gives information on the platform services that we use to assist in deploying and monitoring applications.
+
+## Pre requisites
+
+You need to complete the [set up your Mac for development work][setup-ons-mac]. If this has not been completed, please click on the link and follow the instructions. Afterwards, return back to this page.
+
+Run `dp remote allow develop` and `dp remote allow concourse` to ensure you can access concourse and the develop environment.
+Please note that if you have production access, you can run `dp remote allow production` to access production environment.
+
+[Cloud Technologies and Infrastructure](../services/INFRASTRUCTURE.md) - You should have an understanding of AWS and the infrastructure as the platform services are very closely related to this
+
+## Structure
+![](images/platform-structure.png)
```

Comment:
Should this image contain services like logstash and filebeat that run on ASG nodes to facilitate platform services like kibana?